<section>
    <h3 class="section-subtitle">
        Our History
    </h3>
    <div class="section-text">
        The COVID-19 pandemic has popularised the concept and the use of point-of-care testing kits. Inspired by this,
        our team is attempting to develop a novel point-of-care testing kit that is versatile to detect different types
        of antibiotic resistance. We initially identified a potential tool that might help us in constructing our kit,
        which is the 8-17 deoxyribozymes (DNAzymes), together with the stem-loops and fluoroprobes-containing
        oligonucleotides. These DNAzymes are capable of receiving inputs at their stem-loops, which will then expose
        their catalytic sites, and cause cleavage of the oligonucleotides to produce fluorescence [1]. Using this
        biological logic gate system, we were planning to utilise the DNAzymes stem-loops to detect resistance genes.
        However, upon consulting the ideas with our Principal Investigators, we realised that a detection kit that
        depends on fluorescence-based results is not the ideal choice for constructing a point-of-care diagnostic kit.
        Thus, we shifted our focus to develop a kit that generates colourimetric results instead.
    </div>
</section>

<section>
    <h3 class="section-subtitle">
        Background
    </h3>
    <h3 class="section-subtitle-orange-itatlic">
        Antibiotic Resistance
    </h3>
    <div class="section-text">
        Antibiotics, used for over a century to treat bacterial infections, are one of the most successful medical
        interventions. However, antibiotic resistance has increased due to overuse, causing previously manageable
        diseases
        to resurface. Many infectious diseases, which were wiped out by antibiotics, have become a more severe problem
        in
        recent years. <br>
        <div class="white-space"></div>
        In a study published in 2022, it was estimated that 4.95 millions (95% CI 3.62-6.57 millions) of deaths were
        associated with bacterial antimicrobial resistance (AMR) in 2019, and 1.27 millions of it were caused by
        bacterial
        AMR [2]. These figures indicate that deaths due to AMR represent a major global health concern, comparable in
        scale
        to leading causes of mortality such as ischemic heart disease, stroke, and chronic obstructive pulmonary disease
        [3].
        <br>
        <div class="white-space"></div>
        Out of the 4.95 millions of bacterial AMR-associated deaths, 4.3 millions are occurring in low-income and
        middle-income countries (LMICs) [4]. This indicates a necessary step up in LMICs’ infection prevention and
        control,
        as well as access to affordable diagnosis. According to the Lancet Series on Antimicrobial Resistance, if
        interventions are implemented to align infection prevention and control standards in LMICs’ healthcare settings
        with
        the standards in high-income countries, we will be able to prevent up to 337,000 AMR-associated deaths annually
        [4].
        <br>
        <div class="white-space"></div>

        The rise of antibiotic resistant trends is mainly driven by the high selection pressure caused by the increasing
        use
        and misuse of antibiotics over the years. Over the past two decades, global antimicrobial use has increased by
        46%
        [3]. The usage of antibiotics to cure infections has actually provided advantages for those bacteria that have
        resistance, causing the further spread of resistant bacteria since they survive better in the environment
        compared
        to the susceptible ones [3].

        <br>
        <div class="white-space"></div>
        In the natural process, horizontal gene transfer might also occur among species of bacteria. This process
        enables
        bacteria to exchange their genetic material, including antibiotic resistance genes (ARGs), across diverse
        species,
        which greatly facilitates collaboration among bacterial populations in the development of multidrug resistance
        (MDR)
        [5]. Moreover, MDR leads to the escalation in severity of the problem. The most evident instance is Superbugs, a
        strain of bacteria that are resistant to several types of antibiotics, which greatly challenges the treatment
        process of a bacterial infection.
        <br>
        <div class="white-space"></div>
        On a larger scope, the problem of AMR causes larger problems than just increased challenge in treating bacterial
        infection. It also complicates the treatment of chronic kidney disease, diabetes and associated urinary tract
        and foot infections, chronic obstructive pulmonary disease, liver cirrhosis, as well as surgical care due to the
        risk of exposure to resistant infections [3]. In addition, AMR impedes progress towards the global childhood
        survival rate targeted by the Sustainable Development Goal 3. This is because one third of newborn deaths are
        caused by infections, and half of these infection-related deaths are specifically attributed to sepsis [3].
        Concerningly, the pathogens driving these deadly infections are becoming increasingly resistant to the most
        widely used antibiotic treatments [3].
        <br>
        <div class="white-space"></div>

        Furthermore, the rise of antimicrobial resistance has diminished the effectiveness of previously useful
        antimicrobial drugs, causing access to antimicrobials that were previously considered second-line treatments has
        become increasingly crucial [3]. The Organization for Economic Cooperation and Development (OECD) projects a
        twofold surge in resistance to last-resort antibiotics by 2035 compared to 2005 levels [6].

        <br>
        <div class="white-space"></div>
        In addressing the problems of antibiotic resistance, the Lancet series in 2024 proposed three goals to be
        achieved globally by 2030; 10% decrease in deaths from AMR, 20% reduction in inappropriate human antibiotic use,
        and 30% reduction in inappropriate animal antibiotic use [7]. These goals can only be achieved with optimised
        surveillance, including diagnosis to prevent the misuse of ineffective antibiotics which only further drive
        resistance. However, current technologies for diagnosing antibiotic resistance, such as minimum inhibitory
        concentration (MIC), require lengthy culturing, while genetic tests like PCR necessitate trained personnel.
        Additionally, routine bacteriological culture testing is available in only 1.3% of laboratories in sub-Saharan
        Africa [8]. To counteract the rising trend of antibiotic resistance, and to provide better access to diagnostic
        kits in low-resource settings, a cheaper, faster, and simpler technology is needed.
        <br>
        <div class="white-space"></div>
        <h3 class="section-subtitle-orange-itatlic">
            <em>Klebsiella pneumoniae</em> 
        </h3>
        <em>Klebsiella pneumoniae</em> has often been linked to causing pneumonia [9]. It is a
        gram-negative, encapsulated, and non-motile type of bacterium found in the environment, mainly colonising human
        mucosal surfaces of the oropharynx and gastrointestinal (GI) tract [9]. <em>Klebsiella pneumoniae</em> infection is best treated with third- and fourth-generation cephalosporins, quinolones, or
        carbapenems [10]. However, over the years, <em>Klebsiella pneumoniae</em> has been shown to
        produce extended-spectrum beta-lactamase (ESBL) that breaks down and inactivates beta-lactam antibiotics,
        including cephalosporin, rendering third-generation cephalosporin drugs ineffective for treating <div
            class="itatlic">Klebsiella pneumoniae </div>infections.
        <br>
        <div class="white-space"></div>
        Additionally, cases of carbapenem-resistant <em>Klebsiella pneumoniae</em> have also been
        discovered. This thus classifies Klebsiella pneumonia as one of the Carbapenemase-producing Enterobacteriaceae
        (CPE). Similar to beta-lactamase, carbapenem resistance is conferred by the production of the carbapenemase
        enzyme which is capable of hydrolyzing and disabling the drug molecule [11], [12]. More concerningly,
        carbapenemases are encoded on plasmids, for instance, as bla<i>KPC</i> and bla<i>NDM-1</i> [13]. This means that it can be easily transferred between
        bacteria through horizontal gene transfer, resulting in a more vigorous spread of the resistance [14].
        <br>
        <div class="white-space"></div>
        In the estimated 1.27 million deaths caused by AMR, <em>Klebsiella pneumoniae</em> was one of
        the six leading pathogens [2]. In sub-Saharan Africa, <em>Klebsiella pneumoniae</em> 
        contributed to 19.9% deaths caused by AMR and 17.5% of AMR-associated mortality [2]. Other than that, <div
            class="itatlic">Klebsiella pneumoniae </div>is one of the most common causes of neonatal sepsis in low
        resource settings, such as in sub-Saharan Africa [3], [15]. Although it is estimated that only 3-5% of all
        community-acquired pneumonia in Western culture is related to <em>Klebsiella pneumoniae</em> 
        infection, in developing countries such as African countries, for example, community-acquired <div
            class="itatlic">Klebsiella pneumoniae </div>accounts for roughly 15% of all pneumonia cases [9].
        <br>
        <div class="white-space"></div>
        In terms of its resistant variants, resistant <em>Klebsiella pneumoniae</em> commonly causes
        community-acquired urinary tract infections along with resistant Escherichia coli, Proteus mirabilis, and <div
            class="itatlic">Staphylococcus saprophyticus</div> [3]. According to the World Health Organization, <div
            class="itatlic">Klebsiella pneumoniae </div>showed elevated resistance levels against major antibiotics,
        compromising the effectiveness of these last-resort drugs and increasing the risk of untreatable infections [6].
        This indicates that <em>Klebsiella pneumoniae</em> is one of the most significant pathogens
        when it comes to antibiotic resistance. Additionally, the 2024 Lancet series emphasised on the needs for novel
        strategies to address the burden of ESKAPE pathogens, which includes <em>Klebsiella pneumoniae</em> [4], [13].
        <br>
        <div class="white-space"></div>
        With this regard, we consider <em>Klebsiella pneumoniae</em> as our starting focus in
        developing our diagnosis method. With its level of urgency, especially in low-resource settings, performing
        diagnosis of resistant <em>Klebsiella pneumoniae</em> in a simple and low-cost manner will
        benefit the community for a more effective usage of antibiotics as well as for enhanced surveillance.

    </div>
</section>

<section>
    <h3 class="section-subtitle">
        Our Solution
    </h3>

    <div class="section-text">
        We are developing a point-of-care test kit, ResiSense, for more efficient screening, using cell-free systems.
        This proposed testing system consists of three parts: target gene extraction, rolling circle amplification
        (RCA), and signal expression. <br>
        <div class="white-space"></div>
        After cell lysis releases bacterial DNA, a snippet of the target gene, which is responsible for antibiotic
        resistance, is cut using a pair of restriction enzymes and nickase. The phi-29 polymerase aids in releasing the
        target gene as ss-DNA. This target gene serves as a primer and triggers the formation of peroxidase-mimicking
        DNAzymes through rolling circle amplification (RCA). Finally, the G-quadruplex DNAzymes cause a colour change in
        the solution, indicating the presence of antibiotic-resistant genes in bacteria.
        <br>
        <div class="white-space"></div>
        All processes in our detection system are designed to occur in isothermal conditions, specifically in room
        temperature, and do not require dedicated, large on-site hardware. Our aim is to create a versatile, easily
        programmable system that can detect various antibiotic resistance genes, starting with those related to
        <em>Klebsiella pneumoniae</em> .
    </div>
    <h3 class="section-subtitle-orange-itatlic">
        Choice of Target Bacterium and Gene Fragment
    </h3>
    <div class="section-text">
        <em>Klebsiella pneumoniae</em>  Carbapenemase (KPC) are β-lactamases produced by gram-negative 
    bacteria, responsible for
        the resistance towards penicillins, cephalosporins, monobactams, carbapenems, and other β-lactamase inhibitors
        [16]. KPCs are among the most widespread carbapenemases, with bla<i>KPC-2</i> as the most prevalent gene variant in
        China [17], [18]. Also, the Centre for Health and Protection has reported 2118 cases of <em>Klebsiella pneumoniae</em> 
        infection in 2023, with approximately 111 cases exhibiting extended-spectrum β-lactamase positive (ESBL+)
        phenotype [19]. Hence, we believe that due to the prevalence of KPC in the worldwide and regional context, there
        is a definite need for a test kit for detecting bla<i>KPC-2</i> from <em>Klebsiella pneumoniae</em> .
    </div>


    <img src="/assets/project-description/Project Description.png">
</section>

<section>
    <h3 class="section-subtitle">
        Our Vision
    </h3>

    <div class="section-text">
        As reported in the 2024 Lancet Series on Antimicrobial Resistance, preserving the effectiveness of antibiotics
        will depend on accurate and timely diagnosis of bacterial infections [7]. However, the scaling up of diagnostic
        technologies faces a number of challenges, including technological, economic, and behavioural challenges [7].
        Thus, this project is mainly about developing an affordable, simple, yet accurate diagnostic kit, aimed to
        provide better accessibility to diagnostic technologies in low-resource settings by eliminating the problems of
        lengthy culturing, the need for specialised machines, and trained personnel.<br>
        <div class="white-space"></div>
        After successfully running our systems to detect the resistance genes of <em>Klebsiella pneumoniae</em> , we are planning
        to expand the project to detect resistance genes of other bacterial species, as well as the species itself. We
        understand that diagnosing resistance is insufficient without the information of species. Thus, in the future,
        we will adapt the detection system to also recognize and report the bacteria species that cause the infection.
        <br>
        <div class="white-space"></div>
        All processes in our detection system are designed to occur in isothermal conditions, specifically in room
        temperature, and do not require dedicated, large on-site hardware. Our aim is to create a versatile, easily
        programmable system that can detect various antibiotic resistance genes, starting with those related to
        <em>Klebsiella pneumoniae</em> .


        In the long run, with the lower cost and decent turnaround time, our technology has the potential to help
        doctors choose the right antibiotics to prescribe, saving lives that would not be possible to save without
        access to MALDI-TOF. Our technology also helps prevent the spread of antibiotic resistance by alerting the
        medical professionals of its existence. Financially, we target to open up a new market that was previously
        unreachable with the current technologies. Lastly, we aim to create an impact in research by developing a
        methodology that could be readily adapted to detecting other segments of genes in any organism, helping other
        researchers develop accessible diagnostic tests based on our platform.
    </div>
    
</section>
<h3>References</h3>
<ol class="reference-list">
    <li> M. N. Stojanović and D. Stefanović, “Deoxyribozyme-based half-adder,” <i>Journal of the American Chemical Society</i>, vol. 125, no. 22, pp. 6673–6676, May 2003. doi:10.1021/ja0296632.</li>
        <li> C. J. Murray et al., “Global burden of bacterial antimicrobial resistance in 2019: A systematic analysis,” <i>The Lancet</i>, vol. 399, no. 10325, pp. 629–655, Feb. 2022. doi:10.1016/s0140-6736(21)02724-0.</li>
        <li> I. N. Okeke et al., “The scope of the Antimicrobial Resistance Challenge,” <i>The Lancet</i>, vol. 403, no. 10442, pp. 2426–2438, Jun. 2024. doi:10.1016/s0140-6736(24)00876-6.</li>
        <li> J. A. Lewnard et al., “Burden of bacterial antimicrobial resistance in low-income and middle-income countries avertible by existing interventions: An evidence review and modelling analysis,” <i>The Lancet</i>, vol. 403, no. 10442, pp. 2439–2454, Jun. 2024. doi:10.1016/s0140-6736(24)00862-6.</li>
        <li> D. Sun, K. Jeannot, Y. Xiao, and C. W. Knapp, “Editorial: Horizontal gene transfer mediated bacterial antibiotic resistance,” <i>Frontiers in Microbiology</i>, vol. 10, Aug. 2019. doi:10.3389/fmicb.2019.01933.</li>
        <li> “Antimicrobial resistance,” World Health Organization, <a href="https://www.who.int/news-room/fact-sheets/detail/antimicrobial-resistance#:~:text=Antimicrobials-including%20antibiotics%2C%20antivirals%2C,longer%20respond%20to%20antimicrobial%20medicines"> https://www.who.int/news-room/fact-sheets/detail/antimicrobial-resistance#:~:text=Antimicrobials-including%20antibiotics%2C%20antivirals%2C,longer%20respond%20to%20antimicrobial%20medicines</a> (accessed Apr. 14, 2024).</li>
        <li> “Sustainable Access to Effective Antibiotics: An Executive Summary for The Lancet’s Series,” <i>The Lancet Series on Antimicrobial Resistance</i>, <a href="https://www.thelancet.com/series/antibiotic-resistance?dgcid=tlcom_infographic_amr2024_lancet">https://www.thelancet.com/series/antibiotic-resistance?dgcid=tlcom_infographic_amr2024_lancet</a> (accessed Jun. 17, 2024).</li>
        <li> R. Laxminarayan et al., “Expanding antibiotic, vaccine, and diagnostics development and access to tackle antimicrobial resistance,” <i>The Lancet</i>, vol. 403, no. 10443, pp. 2534–2550, Jun. 2024. doi:10.1016/s0140-6736(24)00878-x.</li>
        <li> J. V. Ashurst and A. Dawson, "Klebsiella Pneumonia," <i>StatPearls</i> [Internet]. Treasure Island (FL): StatPearls Publishing, Jan. 2024. Available: <a href="https://www.ncbi.nlm.nih.gov/books/NBK519004/ ">https://www.ncbi.nlm.nih.gov/books/NBK519004/ </a> (accessed Jun. 18, 2024).</li>
        <li> S. E. Prince, K. A. Dominger, B. A. Cunha, and N. C. Klein, “Klebsiella pneumoniae pneumonia,” <i>Heart & Lung</i>, vol. 26, no. 5, pp. 413–417, Sep./Oct. 1997. doi:10.1016/s0147-9563(97)90028-5.</li>
        <li> L. D. Hughes, A. Aljawadi, and A. Pillai, “An overview of carbapenemase producing Enterobacteriaceae (CPE) in Trauma and Orthopaedics,” <i>Journal of Orthopaedics</i>, vol. 16, no. 6, pp. 455–458, Nov./Dec. 2019. doi:10.1016/j.jor.2019.06.026.</li>
        <li> T. R. Walsh, “Emerging Carbapenemases: A global perspective,” <i>International Journal of Antimicrobial Agents</i>, vol. 36, pp. S8–S14, Nov. 2010. doi:10.1016/s0924-8579(10)70004-2.</li>
        <li> S. Santajit and N. Indrawattana, “Mechanisms of Antimicrobial Resistance in ESKAPE Pathogens,” <i>BioMed Research International</i>, vol. 2016, Jan. 2016. doi:10.1155/2016/2475067.</li>
        <li> “Carbapenemase-producing Enterobacteriaceae (CPE),” National Institute for Communicable Diseases, <a href="https://www.nicd.ac.za/diseases-a-z-index/carbapenemase-producing-enterobacteriaceae-cpe/#:~:text=Carbapenemase%2Dproducing%20Enterobacteriaceae%20(CPE)%20are%20carbapenem%2Dresistant%20Enterobacteriaceae,them%20from%20killing%20the%20bacteria">https://www.nicd.ac.za/diseases-a-z-index/carbapenemase-producing-enterobacteriaceae-cpe/#:~:text=
              Carbapenemase%2Dproducing%20Enterobacteriaceae%20(CPE)%20are%20carbapenem%2Dresistant
              %20Enterobacteriaceae,them%20from%20killing%20the%20bacteria</a> (accessed Jun. 19, 2024).</li>
        <li> U. Okomo et al., “Aetiology of invasive bacterial infection and antimicrobial resistance in neonates in sub-Saharan Africa: A systematic review and meta-analysis in line with the Strobe-ni reporting guidelines,” <i>The Lancet Infectious Diseases</i>, vol. 19, no. 11, pp. 1219–1234, Nov. 2019. doi:10.1016/s1473-3099(19)30414-1.</li>
        <li> L. S. Munoz-Price et al., “Clinical epidemiology of the global expansion of Klebsiella pneumoniae carbapenemases,” <i>The Lancet Infectious Diseases</i>, vol. 13, no. 9, pp. 785–796, Sep. 2013. doi:10.1016/s1473-3099(13)70190-7.</li>
        <li> M. D. Barnes et al., “Klebsiella pneumoniae Carbapenemase-2 (KPC-2), Substitutions at Ambler Position Asp179, and Resistance to Ceftazidime-Avibactam: Unique Antibiotic-Resistant Phenotypes Emerge from β-Lactamase Protein Engineering,” <i>mBio</i>, vol. 8, no. 5, Nov. 2017. doi:10.1128/mbio.00528-17.</li>
        <li> Y. Hu et al., “Prevalence, risk factors and molecular epidemiology of carbapenem-resistant Klebsiella pneumoniae in patients from Zhejiang, China, 2008–2018,” <i>Emerging Microbes & Infections</i>, vol. 9, no. 1, pp. 1771–1779, Jan. 2020. doi:10.1080/22221751.2020.1799721.</li>
        <li> “Bacterial pathogen isolation and percentage of antimicrobial resistance, out-patient setting, 2023,” Centre for Health Protection, Department of Health, The Government of the Hong Kong Special Administrative Region, <a href="https://www.chp.gov.hk/en/statistics/data/10/641/697/7061.html">https://www.chp.gov.hk/en/statistics/data/10/641/697/7061.html</a> (accessed Sep. 24, 2024).</li>
</ol>