ResiSense is a project that aims to provide a cheaper and faster alternative to detecting antibiotic-resistant pathogens via DNA sensing.  

# Problem Background

## Antibiotic Resistance Prevalence

Antibiotic resistance is a growing problem in the world. The [World Health Organization (WHO)](https://www.who.int) has declared antibiotic resistance as **one of the biggest threats to global health, food security, and development** [⤴️](https://www.who.int/news-room/fact-sheets/detail/antimicrobial-resistance). The misuse and overuse of antibiotics have led to the emergence of antibiotic-resistant pathogens. Previously manageable infections resurface as untreatable diseases as **the effectiveness of last-resort drugs are compromised**, leading to increased mortality rates and healthcare costs. Antibiotic resistance directly caused 1.27 million deaths and contributed to 4.95 million deaths in total in 2019 [⤴️](https://www.who.int/news-room/fact-sheets/detail/antimicrobial-resistance), and this figure is projected to rise to 10 million by 2050 if no action is taken [⤴️](https://apo.org.au/sites/default/files/resource-files/2016-05/apo-nid63983.pdf).  

## Current Methods

The current methods for detecting antibiotic-resistant pathogens are **expensive and time-consuming**, limiting its accessibility and potential utility [⤵️](/human-practices#current-methods). A cheaper, faster, and simpler technology is needed.  

# Our Solution

We are developing **a point-of-care test kit, ResiSense**, for more efficient diagnosis, using **cell-free synthetic biology**. This proposed testing system consists of three parts: target gene extraction, rolling circle amplification (RCA) [⤴️](https://en.wikipedia.org/wiki/Rolling_circle_replication), and signal expression. Bacterial DNA is released following cell lysis before a snippet of the target gene responsible for antibiotic resistance is cut using **a pair of restriction enzymes and nickase**. Φ-29 polymerase aids in releasing the target gene as **ss-DNA**. This target gene serves as a primer and triggers the formation of **peroxidase-mimicking (G-quadruplex) DNAzymes** through RCA to cause a **color change in the solution**, indicating the presence of antibiotic-resistant genes in bacteria. Our aim is to create a **versatile, easily programmable system** that can detect various antibiotic resistance genes, starting with those related to ***Klebsiella pneumonia***, which is a common intestinal bacterium that showed elevated resistance levels against critical antibiotics [⤴️](https://www.who.int/news-room/fact-sheets/detail/antimicrobial-resistance).  
![picture 1](<../assets/diagrams/mechanism overview diagram.png>)  

# Expected Results

| Testing Method        | Limitations                                        | Time | Cost (USD) |
| :-------------------- | :------------------------------------------------- | ---: | ---------: |
| MIC                   | Long incubation time                               |  24h |       9.18 |
| PCR                   | Requires specific machines and trained personnel   |  50m |      34.18 |
| Sequencing            | Requires trained personnel and specilised software |  24h |      97.52 |
| MALDI-TOF MS          | Requires expensive equipment and trained personnel |  30m |        0.1 |
| ResiSense (Projected) | Under development                                  |  93m |       5.22 |

Our solution aims to address the problems of lengthy culturing, the need for specilised machines and trained personnel by offering a cheaper, quicker, and simpler technology. We are confident that the innovation will bring a great impact on different aspects.  