<!doctype html>
<html lang="en"
    ><head>
        <meta charset="UTF-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <link rel="icon" type="image/x-icon" href="/assets/favicon.png" />
        <meta base-url="." />
        <title>Modelling</title>

        <link rel="preconnect" href="https://static.igem.wiki" />

        <!-- <link rel="preconnect" href="https://fonts.googleapis.com" /> -->
        <!-- <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin /> -->
        <!-- <link href="https://fonts.googleapis.com/css2?family=Baloo+2:wght@400..800&display=swap" rel="stylesheet" /> -->
        <!-- <link href="https://fonts.googleapis.com/css2?family=Atkinson+Hyperlegible:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet"> -->
        <!-- <link href="https://fonts.googleapis.com/css2?family=Bellota+Text:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap" rel="stylesheet"> -->

        <link rel="stylesheet" type="text/css" href="/styles/globalStylesheet.css" />
        <script type="text/javascript" src="/scripts/globalLibrary.js"></script>
        <link rel="stylesheet" type="text/css" href="/styles/template.css" /><link
            rel="stylesheet"
            type="text/css"
            href="/styles/parallax-doodle.css" /><link
            rel="stylesheet"
            type="text/css"
            href="/styles/header.css" /><link rel="stylesheet" type="text/css" href="/styles/body.css" /><link
            rel="stylesheet"
            type="text/css"
            href="/styles/content.css" /><link rel="stylesheet" type="text/css" href="/styles/footer.css" /><link
            rel="stylesheet"
            type="text/css"
            href="/styles/searchbar.css" /><link
            rel="stylesheet"
            type="text/css"
            href="/styles/hamburger-menu.css" /><link
            rel="stylesheet"
            type="text/css"
            href="/styles/catalogue.css" /><link rel="stylesheet" type="text/css" href="/styles/title-card.css" /><link
            rel="stylesheet"
            type="text/css"
            href="/styles/outline.css" /><link rel="stylesheet" type="text/css" href="/styles/sidebar.css" /><link
            rel="stylesheet"
            type="text/css"
            href="/styles/figures.css" /><link rel="stylesheet" type="text/css" href="/styles/references.css" /><script
            type="text/javascript"
            src="/scripts/responsivenessHandler.js"
            defer=""
        ></script
        ><script type="text/javascript" src="/scripts/parallax-doodle.js" defer=""></script
        ><script type="text/javascript" src="/scripts/content.js" defer=""></script
        ><script type="text/javascript" src="/scripts/searchbar.js" defer=""></script
        ><script type="text/javascript" src="/scripts/hamburger-menu.js" defer=""></script
        ><script type="text/javascript" src="/scripts/catalogue.js" defer=""></script
        ><script type="text/javascript" src="/scripts/title-card.js" defer=""></script
        ><script type="text/javascript" src="/scripts/outline.js" defer=""></script
        ><script type="text/javascript" src="/scripts/references.js" defer=""></script
    ></head>

    <body>
        <painted-template>
            <painted-parallax-doodle>
                <div id="parallax-doodle-container"
                    ><img
                        src="/assets/doodles/red blood cell.svg"
                        style="translate: 57vw 5420px; animation-name: parallax-doodle-2vdgz" /><img
                        src="/assets/doodles/drug.svg"
                        style="translate: 23vw 12620px; animation-name: parallax-doodle-bti5jo" /><img
                        src="/assets/doodles/dna.svg"
                        style="translate: 52vw 160px; animation-name: parallax-doodle-65uwjl" /><img
                        src="/assets/doodles/dna.svg"
                        style="translate: 24vw 16780px; animation-name: parallax-doodle-9ml55j" /><img
                        src="/assets/doodles/red blood cell.svg"
                        style="translate: 42vw 10760px; animation-name: parallax-doodle-hpxtxk" /><img
                        src="/assets/doodles/dna.svg"
                        style="translate: 17vw 7800px; animation-name: parallax-doodle-zqqlte" /><img
                        src="/assets/doodles/virus.svg"
                        style="translate: 55vw 11560px; animation-name: parallax-doodle-bztsi" /><img
                        src="/assets/doodles/dna.svg"
                        style="translate: 6vw 2660px; animation-name: parallax-doodle-qxvh19" /><img
                        src="/assets/doodles/bacteria.svg"
                        style="translate: 99vw 10820px; animation-name: parallax-doodle-6szpdi" /><img
                        src="/assets/doodles/e coli.svg"
                        style="translate: 57vw 10120px; animation-name: parallax-doodle-b2skw" /><img
                        src="/assets/doodles/red blood cell.svg"
                        style="translate: 13vw 9660px; animation-name: parallax-doodle-0at8fo" /><img
                        src="/assets/doodles/drug.svg"
                        style="translate: 76vw 11620px; animation-name: parallax-doodle-heb7el" /><img
                        src="/assets/doodles/drug.svg"
                        style="translate: 28vw 9260px; animation-name: parallax-doodle-zec79n" /><img
                        src="/assets/doodles/virus.svg"
                        style="translate: 96vw 5440px; animation-name: parallax-doodle-6cn9dr" /><img
                        src="/assets/logos/resisense/ResiSense Orange Logo.svg"
                        style="translate: 75vw 8680px; animation-name: parallax-doodle-h3l6wb" /><img
                        src="/assets/doodles/red blood cell.svg"
                        style="translate: 86vw 2980px; animation-name: parallax-doodle-84rjtt" /><img
                        src="/assets/doodles/virus.svg"
                        style="translate: 13vw 15900px; animation-name: parallax-doodle-7fdkth" /><img
                        src="/assets/doodles/bacteria.svg"
                        style="translate: 71vw 20px; animation-name: parallax-doodle-ombvne" /><img
                        src="/assets/doodles/e coli.svg"
                        style="translate: 75vw 14140px; animation-name: parallax-doodle-sofuk" /><img
                        src="/assets/doodles/dna.svg"
                        style="translate: 73vw 9940px; animation-name: parallax-doodle-8166uv" /><img
                        src="/assets/doodles/red blood cell.svg"
                        style="translate: 79vw 7000px; animation-name: parallax-doodle-2mhkq7" /><img
                        src="/assets/doodles/red blood cell.svg"
                        style="translate: 58vw 13280px; animation-name: parallax-doodle-dlupi" /><img
                        src="/assets/doodles/e coli.svg"
                        style="translate: 65vw 5720px; animation-name: parallax-doodle-30qpzo" /><img
                        src="/assets/logos/resisense/ResiSense Orange Logo.svg"
                        style="translate: 67vw 2000px; animation-name: parallax-doodle-5eiynv" /><img
                        src="/assets/doodles/drug.svg"
                        style="translate: 80vw 13840px; animation-name: parallax-doodle-0i0fm" /><img
                        src="/assets/doodles/red blood cell.svg"
                        style="translate: 64vw 15440px; animation-name: parallax-doodle-pgfygk" /><img
                        src="/assets/doodles/bacteria.svg"
                        style="translate: 22vw 3920px; animation-name: parallax-doodle-exgig" /><img
                        src="/assets/doodles/e coli.svg"
                        style="translate: 89vw 15480px; animation-name: parallax-doodle-1dg4jn" /><img
                        src="/assets/logos/resisense/ResiSense Orange Logo.svg"
                        style="translate: 58vw 8880px; animation-name: parallax-doodle-z5l6mf" /><img
                        src="/assets/doodles/drug.svg"
                        style="translate: 78vw 2520px; animation-name: parallax-doodle-xx2pg" /><img
                        src="/assets/doodles/syringe.svg"
                        style="translate: 87vw 14520px; animation-name: parallax-doodle-xf3km" /><img
                        src="/assets/doodles/drug.svg"
                        style="translate: 13vw 4900px; animation-name: parallax-doodle-h234fj" /><img
                        src="/assets/logos/resisense/ResiSense Orange Logo.svg"
                        style="translate: 33vw 6520px; animation-name: parallax-doodle-ez2pna" /><img
                        src="/assets/doodles/dna.svg"
                        style="translate: 0vw 5120px; animation-name: parallax-doodle-dfwitw" /><img
                        src="/assets/doodles/syringe.svg"
                        style="translate: 89vw 12420px; animation-name: parallax-doodle-i3503" /><img
                        src="/assets/doodles/drug.svg"
                        style="translate: 2vw 10920px; animation-name: parallax-doodle-e7vvke" /><img
                        src="/assets/doodles/bacteria.svg"
                        style="translate: 18vw 14180px; animation-name: parallax-doodle-qq29r" /><img
                        src="/assets/doodles/bacteria.svg"
                        style="translate: 94vw 12020px; animation-name: parallax-doodle-vdksvw" /><img
                        src="/assets/doodles/syringe.svg"
                        style="translate: 62vw 10640px; animation-name: parallax-doodle-mdo82" /><img
                        src="/assets/doodles/e coli.svg"
                        style="translate: 47vw 7420px; animation-name: parallax-doodle-xqx3r4" /><img
                        src="/assets/doodles/e coli.svg"
                        style="translate: 55vw 3540px; animation-name: parallax-doodle-xpenqk" /><img
                        src="/assets/doodles/syringe.svg"
                        style="translate: 32vw 16160px; animation-name: parallax-doodle-wzqe7t" /><img
                        src="/assets/doodles/syringe.svg"
                        style="translate: 49vw 640px; animation-name: parallax-doodle-0sfusu" /><img
                        src="/assets/logos/resisense/ResiSense Orange Logo.svg"
                        style="translate: 94vw 9280px; animation-name: parallax-doodle-5eriso" /><img
                        src="/assets/doodles/virus.svg"
                        style="translate: 7vw 13900px; animation-name: parallax-doodle-vetp7v" /><img
                        src="/assets/doodles/bacteria.svg"
                        style="translate: 0vw 360px; animation-name: parallax-doodle-3ordfs" /><img
                        src="/assets/logos/resisense/ResiSense Orange Logo.svg"
                        style="translate: 97vw 4120px; animation-name: parallax-doodle-7covlt" /><img
                        src="/assets/doodles/virus.svg"
                        style="translate: 13vw 8780px; animation-name: parallax-doodle-b7pv5" /><img
                        src="/assets/doodles/bacteria.svg"
                        style="translate: 3vw 11480px; animation-name: parallax-doodle-h36ud9" /><img
                        src="/assets/doodles/virus.svg"
                        style="translate: 24vw 11840px; animation-name: parallax-doodle-1amym" /><img
                        src="/assets/doodles/bacteria.svg"
                        style="translate: 30vw 4540px; animation-name: parallax-doodle-m6g97k" /><img
                        src="/assets/doodles/drug.svg"
                        style="translate: 49vw 8080px; animation-name: parallax-doodle-2unaal" /><img
                        src="/assets/doodles/syringe.svg"
                        style="translate: 95vw 16240px; animation-name: parallax-doodle-lemvdc" /><img
                        src="/assets/doodles/virus.svg"
                        style="translate: 38vw 12480px; animation-name: parallax-doodle-wdje9m" /><img
                        src="/assets/doodles/red blood cell.svg"
                        style="translate: 85vw 900px; animation-name: parallax-doodle-013kdq" /><img
                        src="/assets/doodles/drug.svg"
                        style="translate: 30vw 14640px; animation-name: parallax-doodle-5eoc5" /><img
                        src="/assets/doodles/red blood cell.svg"
                        style="translate: 2vw 15100px; animation-name: parallax-doodle-rqgyyd" /><img
                        src="/assets/doodles/dna.svg"
                        style="translate: 2vw 1320px; animation-name: parallax-doodle-nu89x" /><img
                        src="/assets/logos/resisense/ResiSense Orange Logo.svg"
                        style="translate: 67vw 6520px; animation-name: parallax-doodle-wpk31" /><img
                        src="/assets/doodles/e coli.svg"
                        style="translate: 46vw 3100px; animation-name: parallax-doodle-wkcr6d" /><img
                        src="/assets/doodles/dna.svg"
                        style="translate: 91vw 7960px; animation-name: parallax-doodle-ixhh7b" /><img
                        src="/assets/logos/resisense/ResiSense Orange Logo.svg"
                        style="translate: 51vw 1160px; animation-name: parallax-doodle-hwiaum" /><img
                        src="/assets/doodles/e coli.svg"
                        style="translate: 98vw 7540px; animation-name: parallax-doodle-d3vp2" /><img
                        src="/assets/doodles/syringe.svg"
                        style="translate: 5vw 6780px; animation-name: parallax-doodle-czb0zl" /><img
                        src="/assets/doodles/virus.svg"
                        style="translate: 31vw 5880px; animation-name: parallax-doodle-ic5qxh" /><img
                        src="/assets/logos/resisense/ResiSense Orange Logo.svg"
                        style="translate: 22vw 480px; animation-name: parallax-doodle-3zrjgm" /><img
                        src="/assets/doodles/virus.svg"
                        style="translate: 87vw 180px; animation-name: parallax-doodle-iwoonp" /><img
                        src="/assets/doodles/dna.svg"
                        style="translate: 98vw 17000px; animation-name: parallax-doodle-2w9v2u" /><img
                        src="/assets/doodles/syringe.svg"
                        style="translate: 81vw 17020px; animation-name: parallax-doodle-angm46"
                /></div>
            </painted-parallax-doodle>
            <painted-header>
                <header>
                    <div class="logo-container">
                        <a href="/" class="logo"> </a>
                    </div>
                    <div class="header-shelf">
                        <div class="header-items-container">
                            <div class="header-leftside">
                                <div class="logo-name-text">
                                    <a href="/"> ResiSense </a>
                                </div>
                                <div class="logo-flavour-text"> Finding resistant pathogens one species at a time </div>
                            </div>
                            <div class="page-title-container">
                                <div class="page-title hidden" id="header-page-title">Modelling</div>
                            </div>
                            <div class="header-rightside">
                                <painted-searchbar>
                                    <search id="searchbar" class="header-rightside-button">
                                        <input
                                            id="header-search-field"
                                            type="search"
                                            placeholder="Search... (Ctrl + Shift + F)"
                                            autocomplete="off"
                                        />
                                        <button id="search-button" type="submit">üîç</button>
                                        <dialog id="search-dialog">
                                            <div class="main-search-field-container">
                                                <div class="search-icon">üîç</div>
                                                <input
                                                    id="main-search-field"
                                                    type="search"
                                                    placeholder="Search for a page..."
                                                    autocomplete="off"
                                                />
                                            </div>
                                            <div class="search-results-container">
                                                <div class="tooltips">
                                                    <div class="navigation-tips">
                                                        <div>
                                                            Use <kbd>‚Üë</kbd>, <kbd>‚Üì</kbd>, and <kbd>Enter</kbd> to
                                                            navigate the results.
                                                        </div>
                                                    </div>
                                                    <div class="query-stats">
                                                        Querying took <span id="query-time">0</span>ms. Redrawing took
                                                        <span id="redraw-time">0</span>ms.
                                                    </div>
                                                </div>
                                                <div id="search-results" class="require-scrollbar">
                                                    <template id="search-result-template">
                                                        <a id="search-result">
                                                            <div class="result-leftside">
                                                                <div id="result-title"></div>
                                                                <div id="result-path"></div>
                                                            </div>
                                                            <div class="result-divider"></div>
                                                            <div class="result-rightside">
                                                                <div id="result-content"></div>
                                                            </div>
                                                        </a>
                                                    </template>
                                                    <template id="search-results-end-template">
                                                        <fieldset class="search-results-end">
                                                            <legend>There's nothing more to see here üôà</legend>
                                                        </fieldset>
                                                    </template>
                                                    <template id="prefix-tips-template">
                                                        <div class="prefix-tips">
                                                            <div>Tips:</div>
                                                            <ul>
                                                                <li>
                                                                    Prefix query with <kbd>:</kbd> to only search for
                                                                    content
                                                                </li>
                                                                <li>
                                                                    Prefix query with <kbd>!</kbd> to only search for
                                                                    titles
                                                                </li>
                                                            </ul>
                                                        </div>
                                                    </template>
                                                </div>
                                            </div>
                                        </dialog>
                                    </search>
                                </painted-searchbar>
                                <painted-hamburger-menu>
                                    <button id="hamburger-button" class="header-rightside-button">üçî</button>
                                    <div class="hamburger-menu-container">
                                        <dialog id="hamburger-menu">
                                            <template id="hamburger-menu-item-template">
                                                <div id="hamburger-menu-item">
                                                    <a href=""></a>
                                                </div>
                                            </template>

                                            <div id="hamburger-menu-item">
                                                <a href="/home">home</a>
                                            </div>

                                            <div id="hamburger-menu-item">
                                                <a href="/description">project description</a>
                                            </div>

                                            <div id="hamburger-menu-item">
                                                <a href="/human-practices">integrated human practices</a>
                                            </div>

                                            <div id="hamburger-menu-item">
                                                <a href="/proof-of-concept">proof of concept</a>
                                            </div>

                                            <div id="hamburger-menu-item">
                                                <a href="/engineering">engineering success</a>
                                            </div>

                                            <div id="hamburger-menu-item">
                                                <a href="/education">education</a>
                                            </div>

                                            <div id="hamburger-menu-item" class="hamburger-menu-item-active">
                                                <a href="/model">modelling</a>
                                            </div>

                                            <div id="hamburger-menu-item">
                                                <a href="/results">results</a>
                                            </div>

                                            <div id="hamburger-menu-item">
                                                <a href="/notebook">notebook</a>
                                            </div>

                                            <div id="hamburger-menu-item">
                                                <a href="/contribution">contribution</a>
                                            </div>

                                            <div id="hamburger-menu-item">
                                                <a href="/acknowledgements">acknowledgements</a>
                                            </div>
                                        </dialog>
                                    </div>
                                </painted-hamburger-menu>
                            </div>
                        </div>
                        <div class="header-catalogue-container">
                            <painted-catalogue>
                                <div class="catalogue" id="catalogue">
                                    <template id="catalogue-item-template">
                                        <div class="catalogue-item">
                                            <a href=""></a>
                                        </div>
                                    </template>
                                    <template id="catalogue-divider-template">
                                        <div class="catalogue-item catalogue-divider"><span> | </span></div>
                                    </template>
                                    <!--  -->
                                    <div class="catalogue-item catalogue-arrow"><span> &gt; </span></div>

                                    <div class="catalogue-item">
                                        <a href="/home">home</a>
                                    </div>

                                    <div class="catalogue-item catalogue-divider"><span> | </span></div>

                                    <div class="catalogue-item">
                                        <a href="/description">project description</a>
                                    </div>

                                    <div class="catalogue-item catalogue-divider"><span> | </span></div>

                                    <div class="catalogue-item">
                                        <a href="/human-practices">integrated human practices</a>
                                    </div>

                                    <div class="catalogue-item catalogue-divider"><span> | </span></div>

                                    <div class="catalogue-item">
                                        <a href="/proof-of-concept">proof of concept</a>
                                    </div>

                                    <div class="catalogue-item catalogue-divider"><span> | </span></div>

                                    <div class="catalogue-item">
                                        <a href="/engineering">engineering success</a>
                                    </div>

                                    <div class="catalogue-item catalogue-divider"><span> | </span></div>

                                    <div class="catalogue-item">
                                        <a href="/education">education</a>
                                    </div>

                                    <div class="catalogue-item catalogue-divider"><span> | </span></div>

                                    <div class="catalogue-item catalogue-item-active">
                                        <a href="/model">modelling</a>
                                    </div>

                                    <div class="catalogue-item catalogue-divider"><span> | </span></div>

                                    <div class="catalogue-item">
                                        <a href="/results">results</a>
                                    </div>

                                    <div class="catalogue-item catalogue-divider"><span> | </span></div>

                                    <div class="catalogue-item">
                                        <a href="/notebook">notebook</a>
                                    </div>

                                    <div class="catalogue-item catalogue-divider"><span> | </span></div>

                                    <div class="catalogue-item">
                                        <a href="/contribution">contribution</a>
                                    </div>

                                    <div class="catalogue-item catalogue-divider"><span> | </span></div>

                                    <div class="catalogue-item">
                                        <a href="/acknowledgements">acknowledgements</a>
                                    </div>

                                    <div class="catalogue-item catalogue-divider"><span> | </span></div>
                                </div>
                            </painted-catalogue>
                        </div>
                    </div>
                </header>
                <div class="header-spacer"></div>
            </painted-header>
            <painted-body>
                <div class="body-container">
                    <painted-title-card>
                        <div class="title-card-container">
                            <div class="background-box">
                                <div class="pattern"></div>
                            </div>
                            <div class="page-title use-parallax" id="title-name">Modelling</div>
                            <div class="page-title use-parallax" id="title-name-shadow">Modelling</div>
                            <div class="page-title use-parallax" id="title-name-box">Modelling</div>
                            <div id="sticker-container">
                                <img
                                    class="use-parallax"
                                    id="blue-pills"
                                    src="/assets/stickers/blue pills-Enhanced-SR.png"
                                />
                                <img
                                    class="use-parallax"
                                    id="colourful-pills"
                                    src="/assets/stickers/colourful pills-Enhanced-SR.png"
                                />
                                <img
                                    class="use-parallax"
                                    id="hold-pills"
                                    src="/assets/stickers/hold pills-Enhanced-SR.png"
                                />
                                <img
                                    class="use-parallax"
                                    id="orange-pills"
                                    src="/assets/stickers/orange pills-Enhanced-SR.png"
                                />
                                <img class="use-parallax" id="globe" src="/assets/stickers/globe-Enhanced-SR.png" />
                                <img class="use-parallax" id="gurney" src="/assets/stickers/gurney-Enhanced-SR.png" />
                            </div>
                        </div>
                    </painted-title-card>
                    <div class="main-container">
                        <painted-outline>
                            <nav>
                                <div id="outline-progress-pointer">‚û°Ô∏è</div>
                                <div class="outline">
                                    <ul id="outline-tree">
                                        <template id="outline-heading-template">
                                            <li>
                                                <a>heading</a>
                                                <ul></ul>
                                            </li>
                                        </template>

                                        <li>
                                            <a href="#introduction" id="--outline-introduction">Introduction</a>
                                            <ul></ul>
                                        </li>

                                        <li>
                                            <a href="#methodology" id="--outline-methodology">Methodology</a>
                                            <ul>
                                                <li>
                                                    <a href="#topology-preparation" id="--outline-topology-preparation"
                                                        >Topology Preparation</a
                                                    >
                                                    <ul></ul>
                                                </li>

                                                <li>
                                                    <a
                                                        href="#topology-verification"
                                                        id="--outline-topology-verification"
                                                        >Topology Verification</a
                                                    >
                                                    <ul></ul>
                                                </li>

                                                <li>
                                                    <a
                                                        href="#choice-of-force-field"
                                                        id="--outline-choice-of-force-field"
                                                        >Choice of Force Field</a
                                                    >
                                                    <ul>
                                                        <li>
                                                            <a
                                                                href="#dmso-parametrisation"
                                                                id="--outline-dmso-parametrisation"
                                                                >DMSO Parametrisation</a
                                                            >
                                                            <ul></ul>
                                                        </li>

                                                        <li>
                                                            <a
                                                                href="#haem-parametrisation"
                                                                id="--outline-haem-parametrisation"
                                                                >Haem Parametrisation</a
                                                            >
                                                            <ul></ul>
                                                        </li>
                                                    </ul>
                                                </li>

                                                <li>
                                                    <a href="#topology-assembly" id="--outline-topology-assembly"
                                                        >Topology Assembly</a
                                                    >
                                                    <ul></ul>
                                                </li>

                                                <li>
                                                    <a href="#system-equilibration" id="--outline-system-equilibration"
                                                        >System Equilibration</a
                                                    >
                                                    <ul></ul>
                                                </li>

                                                <li>
                                                    <a href="#production-md" id="--outline-production-md"
                                                        >Production MD</a
                                                    >
                                                    <ul></ul>
                                                </li>

                                                <li>
                                                    <a
                                                        href="#calculation-of-g-of-g-quad-haem-bond"
                                                        id="--outline-calculation-of-g-of-g-quad-haem-bond"
                                                        >Calculation of ŒîG of G-quad-haem Bond</a
                                                    >
                                                    <ul></ul>
                                                </li>
                                            </ul>
                                        </li>

                                        <li>
                                            <a href="#results" id="--outline-results">Results</a>
                                            <ul></ul>
                                        </li>

                                        <li>
                                            <a href="#discussions" id="--outline-discussions">Discussions</a>
                                            <ul></ul>
                                        </li>

                                        <li>
                                            <a
                                                href="#limitations-and-uncertainties"
                                                id="--outline-limitations-and-uncertainties"
                                                >Limitations and Uncertainties</a
                                            >
                                            <ul>
                                                <li>
                                                    <a
                                                        href="#inability-to-simulate-peroxidase-activity-of-g-quad-haem-in-various-solvent-mixtures"
                                                        id="--outline-inability-to-simulate-peroxidase-activity-of-g-quad-haem-in-various-solvent-mixtures"
                                                        >Inability to Simulate Peroxidase Activity of G-quad-haem in
                                                        Various Solvent Mixtures</a
                                                    >
                                                    <ul></ul>
                                                </li>

                                                <li>
                                                    <a
                                                        href="#unrestrained-g-quad-skews-free-energy-values"
                                                        id="--outline-unrestrained-g-quad-skews-free-energy-values"
                                                        >Unrestrained G-quad Skews Free Energy Values</a
                                                    >
                                                    <ul></ul>
                                                </li>

                                                <li>
                                                    <a
                                                        href="#binding-affinity-may-not-be-a-universal-indicator-to-g-quad-haemin-peroxidase-activity"
                                                        id="--outline-binding-affinity-may-not-be-a-universal-indicator-to-g-quad-haemin-peroxidase-activity"
                                                        >Binding Affinity may not be a Universal Indicator to
                                                        G-quad-haemin Peroxidase Activity</a
                                                    >
                                                    <ul></ul>
                                                </li>

                                                <li>
                                                    <a
                                                        href="#g-quad-haem-activity-may-not-be-the-most-crucial-factor-affecting-reaction-rate"
                                                        id="--outline-g-quad-haem-activity-may-not-be-the-most-crucial-factor-affecting-reaction-rate"
                                                        >G-quad-haem Activity may not be the Most Crucial Factor
                                                        Affecting Reaction Rate</a
                                                    >
                                                    <ul></ul>
                                                </li>
                                            </ul>
                                        </li>

                                        <li>
                                            <a href="#conclusion" id="--outline-conclusion">Conclusion</a>
                                            <ul></ul>
                                        </li>

                                        <li>
                                            <a href="#references" id="--outline-references">References</a>
                                            <ul></ul>
                                        </li>
                                    </ul>
                                </div>
                            </nav>
                        </painted-outline>
                        <main>
                            <painted-content
                                ><p
                                    >This page details our dry lab work on the G-quad-haemin complex. To download our
                                    various resources, see the
                                    <a href="/resources#modelling-resources">resources page</a>.
                                </p>
                                <div class="scroll-marker" id="introduction"></div
                                ><section
                                    ><h1 id="introduction">Introduction</h1
                                    ><p
                                        >In our project, G-quadruplex DNAzyme (G-quad) is the
                                        <strong>last step in our detection mechanism</strong>, which produces a
                                        <strong>visible colour change</strong> when the target gene is detected. The
                                        colour change is due to the <strong>peroxidase-mimicking activity</strong> when
                                        the <strong>G-quadruplex DNAzyme is bound with haemin</strong> (G-quad-haemin),
                                        which catalyses the
                                        <strong>oxidation of TMB (colourless) to TMB‚Å∫ (blue)</strong> in the presence of
                                        H‚ÇÇO‚ÇÇ, as shown in <em><a href="#equation-1">Equation 1</a></em>
                                        <a href="#reference-1">[1]</a>. The literature has shown that
                                        <strong
                                            >the binding of haemin to G-quad is crucial to the catalytic
                                            activity</strong
                                        >
                                        of G-quad <a href="#reference-2">[2]</a>, and G-quadruplex is
                                        <strong
                                            >more stable in a mixture of water and dimethyl sulphoxide (DMSO)</strong
                                        >
                                        <a href="#reference-3">[3]</a>. </p
                                    ><p
                                        >Since two of the reagents in the reaction, haemin and TMB, also have better
                                        solubility in DMSO than in water <a href="#reference-4">[4]</a>,
                                        <a href="#reference-5">[5]</a>, it is postulated that
                                        <strong
                                            >adding more DMSO to the reaction mixture may help G-quad work
                                            faster</strong
                                        >. To investigate this, a <strong>molecular dynamics (MD)</strong> approach has
                                        been developed to
                                        <strong
                                            >calculate the binding energy of haemin to G-quad at different
                                            concentrations of DMSO</strong
                                        >
                                        in water. </p
                                    ><blockquote id="equation-1">
                                        <p
                                            ><img
                                                src="/assets/modelling-data/g-quad-tmb-oxidation.svg"
                                                alt="Equation 1: Oxidation of TMB into TMB‚Å∫ by G-quad-haemin complex"
                                            />
                                        </p>
                                        <p
                                            >Equation 1: <em>Oxidation of TMB into TMB‚Å∫ by G-quad-haemin complex</em>
                                        </p> </blockquote
                                    ><div class="scroll-marker" id="methodology"></div
                                ></section>

                                <section
                                    ><h1 id="methodology">Methodology</h1
                                    ><p
                                        >To obtain the <strong>free energy of binding of haemin to G-quad</strong>,
                                        <strong>alchemical transformation</strong> of the haemin is performed, and the
                                        free energy of binding is calculated according to the
                                        <strong>thermodynamic cycle</strong>, as shown in
                                        <em><a href="#figure-1">Figure 1</a></em
                                        >. The software being used is <strong>GROMACS</strong>
                                        <a href="#reference-6">[6]</a>. </p
                                    ><blockquote id="figure-1">
                                        <p
                                            ><img
                                                src="/assets/modelling-data/thermodynamic-cycle-1.svg"
                                                alt="Figure 1: Thermodynamic cycle for calculating the binding energy of haemin to G-quad"
                                            />
                                        </p>
                                        <p
                                            ><img
                                                src="/assets/modelling-data/thermodynamic-cycle-2.svg"
                                                alt="Figure 1: Thermodynamic cycle for calculating the binding energy of haemin to G-quad"
                                            />
                                        </p>
                                        <p
                                            >Figure 1:
                                            <em
                                                >Thermodynamic cycle for calculating the binding energy of haemin to
                                                G-quad</em
                                            ></p
                                        > </blockquote
                                    ><div class="scroll-marker" id="topology-preparation"></div
                                    ><section
                                        ><h2 id="topology-preparation">Topology Preparation</h2
                                        ><p
                                            >The topology is prepared using the <strong>AlphaFold 3</strong> model
                                            <a href="#reference-7">[7]</a>, with
                                            <strong>one DNA input</strong> (<code>5'-GGGTGGGTGGGTGGGTC-3'</code>) and
                                            <strong>one ligand input</strong> (HEM - Haem). It is noted that haemin and
                                            haem are different, with
                                            <strong>haemin having an extra chloride ion</strong> coordinated with the
                                            iron ion (<em><a href="#figure-2">Figure 2</a></em
                                            >). Since haemin and haem have very similar structures, we believe that
                                            <strong>haem is a sufficient approximation to haemin</strong>. The model
                                            generated by AlphaFold 3 with the <strong>highest confidence</strong> is
                                            taken as the topology for simulation. </p
                                        ><blockquote id="figure-2">
                                            <p
                                                ><img
                                                    src="/assets/modelling-data/haemin.svg"
                                                    alt="Figure 2: Structure of haemin (left) and structure of haem (right)"
                                                />
                                                <img
                                                    src="/assets/modelling-data/haem.svg"
                                                    alt="Figure 2: Structure of haemin (left) and structure of haem (right)"
                                                />
                                            </p>
                                            <p
                                                >Figure 2:
                                                <em>Structure of haemin (left) and structure of haem (right)</em>
                                            </p> </blockquote
                                        ><div class="scroll-marker" id="topology-verification"></div></section
                                    ><section
                                        ><h2 id="topology-verification">Topology Verification</h2
                                        ><p
                                            >We have <strong>visually inspected</strong> the model generated to ensure
                                            that AlphaFold 3 provided a reasonable starting topology for simulation. It
                                            was observed that the generated model follows the
                                            <strong>end-stacking binding mode</strong> reported in the literature
                                            <a href="#reference-8">[8]</a>, as shown in
                                            <em><a href="#figure-3">Figure 3</a></em> and
                                            <em><a href="#figure-4">Figure 4</a></em
                                            >. </p
                                        ><blockquote id="figure-3">
                                            <p
                                                ><img
                                                    src="/assets/modelling-data/g-quad.png"
                                                    alt="Figure 3: Folded G-quadruplex DNAzyme generated by AlphaFold 3"
                                            /></p>
                                            <p
                                                >Figure 3: <em>Folded G-quadruplex DNAzyme generated by AlphaFold 3</em
                                                ><br />Note that the G-quadruplex structure in the middle of the DNAzyme
                                                could be clearly seen.
                                            </p> </blockquote
                                        ><blockquote id="figure-4">
                                            <p
                                                ><img
                                                    src="/assets/modelling-data/g-quad-haem.png"
                                                    alt="Figure 4: Folded G-quadruplex DNAzyme bound with haem"
                                            /></p>
                                            <p
                                                >Figure 4: <em>Folded G-quadruplex DNAzyme bound with haem</em
                                                ><br />Note that the haem stacks on top of the G-quadruplex, which is
                                                consistent with the end-stacking binding mode.
                                            </p> </blockquote
                                        ><div class="scroll-marker" id="choice-of-force-field"></div></section
                                    ><section
                                        ><h2 id="choice-of-force-field">Choice of Force Field</h2
                                        ><p
                                            >The <strong>amber14sb_OL15 force field</strong>
                                            <a href="#reference-9">[9]</a>, <a href="#reference-10">[10]</a> is chosen
                                            as the force field to be used in the simulation, as this force field has
                                            been found to
                                            <strong>better resemble experimental measurements</strong> than other force
                                            fields, which would provide a more realistic simulation
                                            <a href="#reference-11">[11]</a>. </p
                                        ><p
                                            >Since DMSO and haem are <strong>not "native" molecular species</strong> to
                                            the force field, they have to be parametrised before the simulation could
                                            proceed. </p
                                        ><div class="scroll-marker" id="dmso-parametrisation"></div
                                        ><section
                                            ><h3 id="dmso-parametrisation">DMSO Parametrisation</h3
                                            ><p
                                                >The DMSO molecule was parametrised by the
                                                <strong>ACPYPE server</strong> <a href="#reference-12">[12]</a>,
                                                <a href="#reference-13">[13]</a>. The topology of DMSO was submitted to
                                                the server as a <code>.pdb</code> file, and the server produces the
                                                <strong><code>.itp</code> and <code>.gro</code> file</strong> required
                                                for building the topology. </p
                                            ><div class="scroll-marker" id="haem-parametrisation"></div></section
                                        ><section
                                            ><h3 id="haem-parametrisation">Haem Parametrisation</h3
                                            ><p
                                                >Since the center of haem contains an <strong>Fe¬≥‚Å∫ ion</strong>, haem
                                                cannot be parametrised by the ACPYPE server as the
                                                <strong
                                                    >Generalised Amber Force Field (GAFF) does not contain the Fe¬≥‚Å∫
                                                    ion</strong
                                                >
                                                <a href="#reference-14">[14]</a>. To parametrise haem in a way that is
                                                compatible with amber14sb_OL15 force field, the
                                                <strong
                                                    >Metal Center Parameter Builder (<a
                                                        href="https://pubs.acs.org/doi/10.1021/acs.jcim.5b00674"
                                                        target="_blank"
                                                        rel="noopener noreferrer"
                                                        ><code>MCPB.py</code></a
                                                    >)</strong
                                                >
                                                is used for preparing the topology <a href="#reference-15">[15]</a>. </p
                                            ><p
                                                >First, the haem molecule is passed to <code>MCPB.py</code> to obtain
                                                input files for <strong>GAMESS-US</strong>
                                                <a href="#reference-16">[16]</a>. Then, quantum calculations were done
                                                using GAMESS-US in order to <strong>optimise the geometry</strong> and
                                                <strong>compute the charge density</strong>. The wave function is
                                                computed using <em><em>B3LYP/6-31G</em> level of theory</em>*, which is
                                                sufficient for haem <a href="#reference-17">[17]</a>.
                                                <strong>Restrained Electrostatic Potential (RESP)</strong>
                                                <a href="#reference-18">[18]</a> was applied to compute the
                                                <strong>partial charges on different atoms</strong>, and the
                                                <strong>Seminario method</strong> <a href="#reference-19">[19]</a> was
                                                used to <strong>compute the bonded parameters</strong>. Lastly, the
                                                <code>.frcmod</code> file were converted to <code>.itp</code> and
                                                <code>.gro</code> using the LEAP program and ACPYPE. </p
                                            ><div class="scroll-marker" id="topology-assembly"></div></section></section
                                    ><section
                                        ><h2 id="topology-assembly">Topology Assembly</h2
                                        ><p
                                            >While DMSO and haem were parametrised in previous steps, they are
                                            <strong>not a part of the residue database</strong>. Therefore, they have to
                                            be handled separately from the G-quad. </p
                                        ><p
                                            >First, the <code>.pdb</code> file generated by AlphaFold 3 was
                                            <strong>separated into two <code>.pdb</code> files</strong>, one for G-quad
                                            and one for haem. The G-quad is
                                            <strong>converted into GROMACS topology</strong> using the
                                            <code>pdb2gmx</code> command. For the haem, the
                                            <code>editconf</code> command is used to convert <code>.pdb</code> to
                                            <code>.gro</code>. The <code>.gro</code> coordinates were manually copied to
                                            the <code>.gro</code> file of the G-quad <code>.gro</code> file. The
                                            <code>.top</code> file was accordingly modified to
                                            <strong>include the topology for haem and the restraints for it</strong
                                            >. </p
                                        ><p
                                            >After that, if <strong>DMSO is used as a co-solvent</strong>, the
                                            <code>insert-molecules</code> command is invoked to
                                            <strong>add DMSO molecules into the topology</strong>.
                                            <code>solvate</code> is used to <strong>add water to the system</strong>.
                                            The <strong>SPC/E water model</strong> <a href="#reference-20">[20]</a> is
                                            used. To neutralise charges, <code>genion</code> is used to
                                            <strong>add ions to the topology</strong>.
                                            <strong>K‚Å∫ ions are added to balance negative charges</strong>, and
                                            <strong>Cl‚Åª ions are added to balance positive charges</strong>, as K‚Å∫ ions
                                            are present in the experimental setup and were also shown to have
                                            <strong>the best effect on stabilising G-quad structures</strong>
                                            <a href="#reference-21">[21]</a>. </p
                                        ><div class="scroll-marker" id="system-equilibration"></div></section
                                    ><section
                                        ><h2 id="system-equilibration">System Equilibration</h2
                                        ><p
                                            >To arrive at a <strong>reasonable starting structure</strong> and to have
                                            <strong>correct sampling</strong> during the production MD, the system must
                                            be well-equilibrated before the production MD runs.
                                            <strong>Energy minimisation</strong> was performed on the solvated structure
                                            using the <strong>steepest gradient descent method</strong>
                                            <a href="#reference-22">[22]</a>. <strong>NVT equilibration</strong> was
                                            performed on the energy-minimised structure. No additional thermostats were
                                            necessary since the <strong>Langevin dynamics integrator</strong> was used
                                            for the simulation, which could also
                                            <strong>regulate the system temperature</strong>. The
                                            <strong>system temperature was set at 298K</strong>.
                                            <strong>NPT equilibration</strong> was performed on the NVT ensemble, using
                                            the <strong>Parrinello-Rahman barostat</strong>
                                            <a href="#reference-23">[23]</a> to
                                            <strong>regulate the system pressure</strong>. The
                                            <strong>pressure is set to be 1 bar</strong>. The NVT and NPT equilibration
                                            were run for 100ps. The <strong>potential</strong>,
                                            <strong>temperature</strong>, <strong>pressure</strong>, and
                                            <strong>density</strong> over time were plotted, and
                                            <strong>convergence was observed</strong> in the graphs. </p
                                        ><div class="scroll-marker" id="production-md"></div></section
                                    ><section
                                        ><h2 id="production-md">Production MD</h2
                                        ><p
                                            >A total of
                                            <strong>41 production MDs were run for each solvent configuration</strong>,
                                            in which the
                                            <strong>charge interaction of haem is gradually decoupled</strong> first,
                                            and then the <strong>Lennard-Jones potentials</strong> are also gradually
                                            decoupled. This allows us to obtain the <strong>‚àÇH/‚àÇŒª curve</strong>, which
                                            can then be used to
                                            <strong>obtain ŒîG using the Bennett Acceptance Ratio (BAR) method</strong>
                                            <a href="#reference-24">[24]</a>. </p
                                        ><p
                                            >The charge lambdas are first decoupled in steps of 0.05 per iteration, and
                                            after the charge interactions are decoupled, the Lennard-Jones potentials
                                            are decoupled in steps of 0.05 per iteration. To
                                            <strong>ensure sufficient ensemble sampling</strong> at the specified lambda
                                            values,
                                            <strong
                                                >2.5ns of the simulation was run for each combination of lambda</strong
                                            >, which is around two times what is deemed adequate
                                            <a href="#reference-25">[25]</a>. The
                                            <strong>free energy was obtained from ‚àÇH/‚àÇŒª</strong> using
                                            <code>bar</code>. </p
                                        ><div
                                            class="scroll-marker"
                                            id="calculation-of-g-of-g-quad-haem-bond"
                                        ></div></section
                                    ><section
                                        ><h2 id="calculation-of-g-of-g-quad-haem-bond"
                                            >Calculation of ŒîG of G-quad-haem Bond</h2
                                        ><p
                                            >To calculate the <strong>ŒîG of haem binding to G-quad</strong>, one may
                                            refer to the <strong>thermodynamic cycle</strong> outlined in
                                            <em><a href="#figure-1">Figure 1</a></em
                                            >. Two separate simulations were set up, one with G-quad bounded with haem
                                            and another with haem only. Two systems were solvated in the
                                            <strong>same DMSO/water mixture concentration</strong>, and haem was
                                            <strong>alchemically transformed</strong> through the process outlined
                                            above. &nbsp;</p
                                        ><div class="scroll-marker" id="results"></div></section
                                ></section>

                                <section
                                    ><h1 id="results">Results</h1
                                    ><p
                                        >The simulation results are summarised below. ŒîG are given in
                                        <strong>kJ/mol</strong> and the +/- values represent the
                                        <strong>standard deviation of ŒîG</strong>.</p
                                    ><table>
                                        <thead>
                                            <tr>
                                                <th>DMSO concentration (v/v) %</th>
                                                <th>ŒîG in G-quad-haem</th>
                                                <th>ŒîG in haem only</th>
                                                <th>ŒîG binding</th>
                                            </tr>
                                        </thead>
                                        <tbody
                                            ><tr>
                                                <td>0</td>
                                                <td>2598.74 +/- 15.45</td>
                                                <td>2633.05 +/- 5.10</td>
                                                <td>34.41 +/- 16.27</td>
                                            </tr>
                                            <tr>
                                                <td>5</td>
                                                <td>2577.23 +/- 3.00</td>
                                                <td>2592.05 +/- 7.48</td>
                                                <td>14.82 +/- 8.06</td>
                                            </tr>
                                            <tr>
                                                <td>10</td>
                                                <td>2577.19 +/- 11.07</td>
                                                <td>2615.68 +/- 2.05</td>
                                                <td>38.49 +/- 11.26</td>
                                            </tr>
                                            <tr>
                                                <td>15</td>
                                                <td>2562.85 +/- 4.34</td>
                                                <td>2604.41 +/- 2.51</td>
                                                <td>41.56 +/- 5.01</td>
                                            </tr>
                                            <tr>
                                                <td>20</td>
                                                <td>2510.37 +/- 17.08</td>
                                                <td>2626.85 +/- 1.66</td>
                                                <td>116.48 +/- 17.16</td>
                                            </tr>
                                            <tr>
                                                <td>25</td>
                                                <td>2543.50 +/- 11.67</td>
                                                <td>2568.09 +/- 2.83</td>
                                                <td>24.59 +/- 12.01</td>
                                            </tr>
                                            <tr>
                                                <td>30</td>
                                                <td>2482.45 +/- 9.99</td>
                                                <td>2600.48 +/- 3.56</td>
                                                <td>118.03 +/- 10.61</td>
                                            </tr>
                                            <tr>
                                                <td>35</td>
                                                <td>2492.22 +/- 13.88</td>
                                                <td>2594.15 +/- 5.82</td>
                                                <td>101.93 +/- 15.05</td>
                                            </tr>
                                            <tr>
                                                <td>40</td>
                                                <td>2564.71 +/- 22.22</td>
                                                <td>2586.21 +/- 3.90</td>
                                                <td>21.50 +/- 22.56</td>
                                            </tr>
                                            <tr>
                                                <td>45</td>
                                                <td>2479.50 +/- 18.14</td>
                                                <td>2558.66 +/- 3.21</td>
                                                <td>79.16 +/- 18.42</td>
                                            </tr>
                                            <tr>
                                                <td>50</td>
                                                <td>2509.47 +/- 15.19</td>
                                                <td>2524.21 +/- 9.77</td>
                                                <td>14.74 +/- 18.06</td>
                                            </tr>
                                        </tbody></table
                                    ><p
                                        >To visualise the results, the
                                        <strong>free energies are plotted against DMSO concentration (v/v) %</strong>,
                                        as shown in <em><a href="#figure-5">Figure 5</a></em> and
                                        <em><a href="#figure-6">Figure 6</a></em
                                        >.</p
                                    ><blockquote id="figure-5" class="full">
                                        <p
                                            ><img
                                                src="/assets/modelling-data/decouple-delta-g.svg"
                                                alt="Figure 5: Change in free energy (ŒîG) of decoupling of haem in G-quad-haem and haem only"
                                            />
                                        </p>
                                        <p
                                            >Figure 5:
                                            <em
                                                >Change in free energy (ŒîG) of decoupling of haem in G-quad-haem and
                                                haem only</em
                                            >
                                        </p> </blockquote
                                    ><blockquote id="figure-6" class="full">
                                        <p
                                            ><img
                                                src="/assets/modelling-data/binding-delta-g.svg"
                                                alt="Figure 6: Change in free energy (ŒîG) of binding of haem in G-quad-haem"
                                            />
                                        </p>
                                        <p
                                            >Figure 6:
                                            <em>Change in free energy (ŒîG) of binding of haem in G-quad-haem</em>
                                        </p> </blockquote
                                    ><div class="scroll-marker" id="discussions"></div
                                ></section>

                                <section
                                    ><h1 id="discussions">Discussions</h1
                                    ><p
                                        >From <em><a href="#figure-5">Figure 5</a></em
                                        >, we could observe that the
                                        <strong>free energy change of decoupling</strong> of haem only in solvent
                                        decreases as DMSO concentration increases. This is in agreement with the
                                        literature, where the
                                        <strong>solubility of haemin increases as DMSO concentration increases</strong>.
                                        In <em><a href="#figure-6">Figure 6</a></em
                                        >, we observed a general trend of <strong>increasing binding energy</strong> of
                                        haem to G-quad, indicating a prediction of
                                        <strong
                                            >decreasing binding affinity to G-quad as the DMSO concentration
                                            increases</strong
                                        >. </p
                                    ><p
                                        ><strong>Some data points do not follow the trend</strong>: the binding energies
                                        at 25% and 40% DMSO concentration (v/v). We believe this is due to the
                                        <strong>change in G-quad conformation during the simulation</strong>, which
                                        <strong>skewed the free energy values</strong>. Through the inspection of
                                        <strong>RMSD value over time</strong> of simulations (see
                                        <a href="/resources#supporting-information">Supporting Information</a>) and
                                        comparing with the <strong>non-perturbed simulation (Œª = 0)</strong>, we
                                        conclude that the
                                        <strong>G-quadruplex DNAzyme has changed conformation</strong> in some of the
                                        simulations, leading to <strong>inaccurate free energy values</strong>.
                                        Potential solutions are discussed
                                        <a href="#unrestrained-g-quad-skews-free-energy-values">below</a>. </p
                                    ><p
                                        >We have also noticed that the
                                        <strong
                                            >standard deviations of the change in free energy for G-quad-haem decoupling
                                            are large</strong
                                        >. Upon inspection of the <code>gmx bar</code> output, we noticed that the
                                        <strong>relative entropies between some of the lambdas were significant</strong
                                        >, indicating <strong>inadequate phase space sampling</strong>. This suggests
                                        the need for more sampling in those intervals. </p
                                    ><div class="scroll-marker" id="limitations-and-uncertainties"></div
                                ></section>

                                <section
                                    ><h1 id="limitations-and-uncertainties">Limitations and Uncertainties</h1
                                    ><p
                                        >While the simulations have provided valuable insights in terms of the affinity
                                        of haem to G-quad, we are also aware of the following limitations and
                                        uncertainties. </p
                                    ><div
                                        class="scroll-marker"
                                        id="inability-to-simulate-peroxidase-activity-of-g-quad-haem-in-various-solvent-mixtures"
                                    ></div
                                    ><section
                                        ><h2
                                            id="inability-to-simulate-peroxidase-activity-of-g-quad-haem-in-various-solvent-mixtures"
                                            >Inability to Simulate Peroxidase Activity of G-quad-haem in Various Solvent
                                            Mixtures</h2
                                        ><p
                                            >While molecular dynamics simulations may provide information about the
                                            <strong>binding affinity between G-quad and haem</strong>, they
                                            <strong
                                                >cannot provide information on the peroxidase activity of
                                                G-quad-haemin</strong
                                            >
                                            due to their
                                            <strong>inability to simulate bond breaking and forming</strong>, which are
                                            the basis of chemical reactions. Such processes would lead to
                                            <strong>topology modifications</strong>, which is
                                            <strong>impossible in pure molecular mechanics simulations</strong>. Also,
                                            in molecular mechanics simulations,
                                            <strong>quantum effects on the electrons are neglected</strong>, and
                                            molecules are <strong>assumed to be at the ground state</strong>. In
                                            contrast,
                                            <strong
                                                >most reactions require electrons to be excited or exchange electrons
                                                between different species</strong
                                            >. Resolving such issues would demand the model to
                                            <strong>take quantum effects into account</strong>. We suggest that
                                            <strong
                                                >quantum mechanics/ molecular mechanics (QM/MM) hybrid
                                                simulations</strong
                                            >
                                            may be able to fill this gap. &nbsp;</p
                                        ><div
                                            class="scroll-marker"
                                            id="unrestrained-g-quad-skews-free-energy-values"
                                        ></div></section
                                    ><section
                                        ><h2 id="unrestrained-g-quad-skews-free-energy-values"
                                            >Unrestrained G-quad Skews Free Energy Values</h2
                                        ><p
                                            >Upon inspection of the G-quad-haem structure generated by the AlphaFold 3
                                            model, we noticed that there are
                                            <strong
                                                >hand-like structures formed by the last two nucleotides of the G-quad
                                                sequence (TC-3')</strong
                                            >
                                            that seem to
                                            <strong>hold onto the haem ligand and keep it in place</strong>. However, as
                                            the interaction between haem and the surroundings was decoupled, the
                                            <strong>interactions between haem and the hand region weakened</strong>,
                                            which may cause the hand region to <strong>wiggle freely</strong>. This
                                            <strong>increases the number of possible microstates</strong> in the system,
                                            <strong>increasing the entropy</strong> and
                                            <strong>leading to skewed free energy values</strong>. This may be solved by
                                            <strong>"confine and release" free energy simulations</strong>, which have
                                            been proposed in literature <a href="#reference-26">[26]</a>. </p
                                        ><div
                                            class="scroll-marker"
                                            id="binding-affinity-may-not-be-a-universal-indicator-to-g-quad-haemin-peroxidase-activity"
                                        ></div></section
                                    ><section
                                        ><h2
                                            id="binding-affinity-may-not-be-a-universal-indicator-to-g-quad-haemin-peroxidase-activity"
                                            >Binding Affinity may not be a Universal Indicator to G-quad-haemin
                                            Peroxidase Activity</h2
                                        ><p
                                            >While there was literature suggesting that
                                            <strong
                                                >the binding affinity of haemin to G-quad may be a predictor for the
                                                peroxidase activity of the G-quad-haemin DNAzyme</strong
                                            >
                                            <a href="#reference-8">[8]</a>, we have also found literature supporting the
                                            opposing view <a href="#reference-27">[27]</a>. Hence, the hypothesis that
                                            the free energy of binding between G-quad and haemin is a predictor for
                                            peroxidase activity <strong>may not be universally true</strong>. </p
                                        ><div
                                            class="scroll-marker"
                                            id="g-quad-haem-activity-may-not-be-the-most-crucial-factor-affecting-reaction-rate"
                                        ></div></section
                                    ><section
                                        ><h2
                                            id="g-quad-haem-activity-may-not-be-the-most-crucial-factor-affecting-reaction-rate"
                                            >G-quad-haem Activity may not be the Most Crucial Factor Affecting Reaction
                                            Rate</h2
                                        ><p
                                            >Although we focused on evaluating the
                                            <strong
                                                >stability of the G-quad-haem complex in different solvent
                                                mixtures</strong
                                            >, one must remember that the ultimate goal for optimising solvent mixture
                                            is to <strong>maximise the reaction rates</strong>, such that the test kit
                                            develops <strong>a more evident colour change more rapidly</strong>. The
                                            stability of the G-quad-haem complex
                                            <strong>only evaluated the catalyst</strong> in the reaction, and the
                                            effects of solvents on the reactants' stability may need to be accounted for
                                            in the simulations. The solvent may increase the reaction rate by
                                            <strong>affecting the stability of intermediate and final products</strong>
                                            and vice versa <a href="#reference-28">[28]</a>. It is suggested that the
                                            whole reaction, including its intermediary products, may be
                                            <strong>qualitatively evaluated in terms of their stability</strong> before
                                            confirming whether the increased or decreased stability of G-quad-haem
                                            matters for the reaction. </p
                                        ><div class="scroll-marker" id="conclusion"></div></section
                                ></section>

                                <section
                                    ><h1 id="conclusion">Conclusion</h1
                                    ><p
                                        >Despite the presence of some inaccuracies due to the factors outlined above,
                                        our preliminary modelling results indicate a promising potential that
                                        <strong
                                            >the binding energy of haem to G-quad is the lowest in fully aqueous
                                            solutions</strong
                                        >. This exciting discovery could pave the way for a new method to
                                        <strong
                                            >predict G-quad-haem complex catalytic capability in different
                                            solutions</strong
                                        >, a possibility that can be further explored in the lab. </p
                                    ><p
                                        >To help future iGEM teams in modelling molecular systems involving the haem
                                        residue, we have uploaded the
                                        <strong
                                            ><a href="/resources#parametrised-haem-topology"
                                                >parametrised haem topology</a
                                            ></strong
                                        >
                                        for other teams to use. </p
                                    ><div class="scroll-marker" id="references"></div
                                ></section>

                                <section
                                    ><h1 id="references">References</h1
                                    ><ol>
                                        <li id="reference-1"
                                            >Z. Zhang, W. Zhao, C. Hu, Y. Cao, Y. Liu, and Q. Liu, "A Convenient and
                                            Label-Free Colourimetric Detection for L-Histidine Based on Inhibition of
                                            Oxidation of 3,3‚Ä≤,5,5‚Ä≤-Tetramethylbenzidine-H‚ÇÇO‚ÇÇ System Triggered by Copper
                                            Ions", vol. 9, p. 773519, Nov. 2021, doi:
                                            <a
                                                href="https://www.doi.org/10.3389/fchem.2021.773519"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.3389/fchem.2021.773519</a
                                            >.</li
                                        >
                                        <li id="reference-2"
                                            >Y. Cao&nbsp;<em>et al.</em>, "Investigation and improvement of catalytic
                                            activity of G-quadruplex/haemin DNAzymes using designed terminal G-tetrads
                                            with deoxyadenosine caps", vol. 11, no. 26, pp. 6896‚Äì696, Jul. 2020, doi:
                                            <a
                                                href="https://www.doi.org/10.1039/d0sc01905d"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1039/d0sc01905d</a
                                            >.</li
                                        >
                                        <li id="reference-3"
                                            >N. Tariq, T. Kume, L. Luo, Z. Cai, S. Dong, and R. B. Macgregor, "Dimethyl
                                            sulphoxide (DMSO) is a stabilising co-solvent for G-quadruplex DNA", vol.
                                            282, p. 106741, Mar. 2022, doi:
                                            <a
                                                href="https://www.doi.org/10.1016/j.bpc.2021.106741"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1016/j.bpc.2021.106741</a
                                            >.</li
                                        >
                                        <li id="reference-4"
                                            >R. Stiebler, A. N. Hoang, T. J. Egan, D. W. Wright, and M. F. Oliveira,
                                            "Increase on the Initial Soluble Haem Levels in Acidic Conditions Is an
                                            Important Mechanism for Spontaneous Haem Crystallisation In Vitro", vol. 5,
                                            no. 9, p. e12694, Sep. 2010, doi:
                                            <a
                                                href="https://www.doi.org/10.1371/journal.pone.0012694"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1371/journal.pone.0012694</a
                                            >.</li
                                        >
                                        <li id="reference-5"
                                            >A. Frey, B. Meckelein, D. Externest, and M. A. Schmidt, "A stable and
                                            highly sensitive 3,3‚Ä≤,5,5‚Ä≤-tetramethylbenzidine-based substrate reagent for
                                            enzyme-linked immunosorbent assays", vol. 233, no. 1, pp. 47‚Äì56, Jan. 2000,
                                            doi:
                                            <a
                                                href="https://www.doi.org/10.1016/S0022-1759(99)00166-0"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1016/S0022-1759(99)00166-0</a
                                            >.</li
                                        >
                                        <li id="reference-6"
                                            >M. Abraham <em>et al.</em>, "GROMACS 2024.3 Source code". Zenodo, 29 Aug,
                                            2024. doi:
                                            <a
                                                href="https://www.doi.org/10.5281/zenodo.13456374"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.5281/zenodo.13456374</a
                                            >.</li
                                        >
                                        <li id="reference-7"
                                            >J. Abramson&nbsp;<em>et al.</em>, "Accurate structure prediction of
                                            biomolecular interactions with AlphaFold 3", vol. 630, no. 8016, pp.
                                            493‚Äì500, Jun. 2024, doi:
                                            <a
                                                href="https://www.doi.org/10.1038/s41586-024-07487-w"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1038/s41586-024-07487-w</a
                                            >.</li
                                        >
                                        <li id="reference-8"
                                            >N. Alizadeh, A. Salimi, and R. Hallaj,&nbsp;<em
                                                >Haemin/G-Quadruplex Horseradish Peroxidase-Mimicking DNAzyme: Principle
                                                and Biosensing Application</em
                                            >, vol. 170. Switzerland: Springer International Publishing AG, 2017, pp.
                                            85‚Äì106.</li
                                        >
                                        <li id="reference-9"
                                            >M. ZgarbovaÃÅ, J. SÃåponer, M. Otyepka, T. E. Cheatham, R. Galindo-Murillo, and
                                            P. JurecÃåka, "Refinement of the Sugar‚ÄìPhosphate Backbone Torsion Beta for
                                            AMBER Force Fields Improves the Description of Z- and B‚ÄëDNA", vol. 11, no.
                                            12, pp. 5723‚Äì5736, Dec. 2015, doi:
                                            <a
                                                href="https://www.doi.org/10.1021/acs.jctc.5b00716"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1021/acs.jctc.5b00716</a
                                            >.</li
                                        >
                                        <li id="reference-10"
                                            >M. ZgarbovaÃÅ&nbsp;<em>et al.</em>, "Refinement of the Cornell et al. Nucleic
                                            Acids Force Field Based on Reference Quantum Chemical Calculations of
                                            Glycosidic Torsion Profiles", vol. 7, no. 9, pp. 2886‚Äì2902, Sep. 2011, doi:
                                            <a
                                                href="https://www.doi.org/10.1021/ct200162x"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1021/ct200162x</a
                                            >.</li
                                        >
                                        <li id="reference-11"
                                            >Li N., Gao Y., Qiu F., and Zhu T., "Benchmark Force Fields for the
                                            Molecular Dynamic Simulation of G-Quadruplexes", vol. 26, no. 17, pp. 5379-,
                                            Sep. 2021, doi:
                                            <a
                                                href="https://www.doi.org/10.3390/molecules26175379"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.3390/molecules26175379</a
                                            >.</li
                                        >
                                        <li id="reference-12"
                                            >Sousa da Silva A. W. and Vranken W. F., "ACPYPE - AnteChamber PYthon Parser
                                            interfacE", vol. 5, no. 1, pp. 367--367, Jul. 2012, doi:
                                            <a
                                                href="https://www.doi.org/10.1186/1756-0500-5-367"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1186/1756-0500-5-367</a
                                            >.</li
                                        >
                                        <li id="reference-13"
                                            >L. Kagami, A. Wilter, A. Diaz, and W. Vranken, "The ACPYPE web server for
                                            small-molecule MD topology generation", vol. 39, no. 6, Jun. 2023, doi:
                                            <a
                                                href="https://www.doi.org/10.1093/bioinformatics/btad350"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1093/bioinformatics/btad350</a
                                            >.</li
                                        >
                                        <li id="reference-14"
                                            >J. Wang, R. M. Wolf, J. W. Caldwell, P. A. Kollman, and D. A. Case,
                                            "Development and testing of a general amber force field", vol. 25, no. 9,
                                            pp. 1157‚Äì1174, Jul. 2004, doi:
                                            <a
                                                href="https://www.doi.org/10.1002/jcc.20035"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1002/jcc.20035</a
                                            >.</li
                                        >
                                        <li id="reference-15"
                                            >P. Li and K. M. Merz, "MCPB.py: A Python Based Metal Center Parameter
                                            Builder", vol. 56, no. 4, pp. 599‚Äì604, Apr. 2016, doi:
                                            <a
                                                href="https://www.doi.org/10.1021/acs.jcim.5b00674"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1021/acs.jcim.5b00674</a
                                            >.</li
                                        >
                                        <li id="reference-16"
                                            >Barca G. M. J.&nbsp;<em>et al.</em>, "Recent developments in the general
                                            atomic and molecular electronic structure system", vol. 152, no. 15, p.
                                            154102, Apr. 2020, doi:
                                            <a
                                                href="https://www.doi.org/10.1063/5.0005188"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1063/5.0005188</a
                                            >.</li
                                        >
                                        <li id="reference-17"
                                            >Autenrieth F., Tajkhorshid E., Baudry J., and Luthey‚ÄêSchulten Z.,
                                            "Classical force field parameters for the haem prosthetic group of
                                            cytochrome c", vol. 25, no. 13, pp. 1613‚Äì1622, Oct. 2004, doi:
                                            <a
                                                href="https://www.doi.org/10.1002/jcc.20079"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1002/jcc.20079</a
                                            >.</li
                                        >
                                        <li id="reference-18"
                                            >C. I. Bayly, P. Cieplak, W. Cornell, and P. A. Kollman, "A well-behaved
                                            electrostatic potential based method using charge restraints for deriving
                                            atomic charges: the RESP model", vol. 97, no. 40, pp. 10269‚Äì10280, Oct.
                                            1993, doi:
                                            <a
                                                href="https://www.doi.org/10.1021/j100142a004"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1021/j100142a004</a
                                            >.</li
                                        >
                                        <li id="reference-19"
                                            >J. M. Seminario, "Calculation of intramolecular force fields from
                                            second-derivative tensors", vol. 60, no. 7, pp. 1271‚Äì1277, 1996, doi:
                                            <a
                                                href="https://www.doi.org/10.1002/(SICI)1097-461X(1996)60:7%3C1271::AID-QUA8%3E3.0.CO;2-W"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1002/(SICI)1097-461X(1996)60:7&lt;1271::AID-QUA8&gt;3.0.CO;2-W</a
                                            >.</li
                                        >
                                        <li id="reference-20"
                                            >H. J. C. Berendsen, J. R. Grigera, and T. P. Straatsma, "The missing term
                                            in effective pair potentials", vol. 91, no. 24, pp. 6269‚Äì6271, Nov. 1987,
                                            doi:
                                            <a
                                                href="https://www.doi.org/10.1021/j100308a038"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1021/j100308a038</a
                                            >.</li
                                        >
                                        <li id="reference-21"
                                            >S. Haider, "Computational Methods to Study G-Quadruplex‚ÄìLigand Complexes",
                                            vol. 98, no. 3, pp. 325‚Äì339, Sep. 2018, doi:
                                            <a
                                                href="https://www.doi.org/10.1007/s41745-018-0083-3"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1007/s41745-018-0083-3</a
                                            >.</li
                                        >
                                        <li id="reference-22"
                                            >E. J. Haug, J. S. Arora, and K. Matsui, "A steepest-descent method for
                                            optimisation of mechanical systems", vol. 19, no. 3, pp. 401‚Äì424, Jul. 1976,
                                            doi:
                                            <a
                                                href="https://www.doi.org/10.1007/BF00941484"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1007/BF00941484</a
                                            >.</li
                                        >
                                        <li id="reference-23"
                                            >Parrinello M. and Rahman A., "Polymorphic transitions in single crystals: A
                                            new molecular dynamics method", vol. 52, no. 12, pp. 7182‚Äì7190, Dec. 1981,
                                            doi:
                                            <a
                                                href="https://www.doi.org/10.1063/1.328693"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1063/1.328693</a
                                            >.</li
                                        >
                                        <li id="reference-24"
                                            >C. H. Bennett, "Efficient estimation of free energy differences from Monte
                                            Carlo data", vol. 22, no. 2, pp. 245‚Äì268, Oct. 1976, doi:
                                            <a
                                                href="https://www.doi.org/10.1016/0021-9991(76)90078-4"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1016/0021-9991(76)90078-4</a
                                            >.</li
                                        >
                                        <li id="reference-25"
                                            >R. S. Rathore, P. Aparoy, P. Reddanna, A. K. Kondapi, and M. Rami Reddy,
                                            "Minimum MD simulation length required to achieve reliable results in free
                                            energy perturbation calculations: Case study of relative binding free
                                            energies of fructose-1,6-bisphosphatase inhibitors", vol. 32, no. 10, pp.
                                            2097‚Äì2103, Jul. 2011, doi:
                                            <a
                                                href="https://www.doi.org/10.1002/jcc.21791"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1002/jcc.21791</a
                                            >.</li
                                        >
                                        <li id="reference-26"
                                            >D. L. Mobley, J. D. Chodera, and K. A. Dill, "Confine-and-Release Method:
                                            Obtaining Correct Binding Free Energies in the Presence of Protein
                                            Conformational Change", vol. 3, no. 4, pp. 1231‚Äì1235, Jul. 2007, doi:
                                            <a
                                                href="https://www.doi.org/10.1021/ct700032n"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1021/ct700032n</a
                                            >.</li
                                        >
                                        <li id="reference-27"
                                            >J. Chen&nbsp;<em>et al.</em>, "How Proximal Nucleobases Regulate the
                                            Catalytic Activity of G‚ÄëQuadruplex/Haemin DNAzymes", vol. 8, no. 12, pp.
                                            11352‚Äì11361, Dec. 2018, doi:
                                            <a
                                                href="https://www.doi.org/10.1021/acscatal.8b03811"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1021/acscatal.8b03811</a
                                            >.</li
                                        >
                                        <li id="reference-28"
                                            >J. T. Hynes, "Chemical Reaction Dynamics in Solution", vol. 36, no. 1, pp.
                                            573‚Äì597, Oct. 1985, doi:
                                            <a
                                                href="https://www.doi.org/10.1146/annurev.pc.36.100185.003041"
                                                target="_blank"
                                                rel="noopener noreferrer"
                                                >10.1146/annurev.pc.36.100185.003041</a
                                            >.</li
                                        >
                                    </ol></section
                                >
                            </painted-content></main
                        >
                        <painted-sidebar>
                            <aside>
                                <a href="#" class="top-button"> üîù </a>
                            </aside>
                        </painted-sidebar>
                    </div>
                </div>
            </painted-body>
            <painted-footer>
                <footer>
                    <div class="footer-banner">
                        <div class="resisense">
                            <div class="resisense-logo">
                                <img src="/assets/logos/resisense/ResiSense Green Logo.svg" alt="ResiSense logo" />
                            </div>
                            <div class="resisense-logo-aside">
                                <div class="resisense-name"> ResiSense </div>
                                <div class="socials">
                                    <div class="social">
                                        <a
                                            href="https://www.instagram.com/cuhkigem2024/"
                                            target="_blank"
                                            rel="noopener noreferrer"
                                        >
                                            <img src="/assets/icons/instagram icon.svg" alt="Instagram icon" />
                                            <span> cuhkigem2024 </span>
                                        </a>
                                    </div>
                                    <div class="social">
                                        <a
                                            href="mailto:cuhkigem2024@gmail.com"
                                            target="_blank"
                                            rel="noopener noreferrer"
                                        >
                                            <img src="/assets/icons/email icon.svg" alt="Email icon" />
                                            <span> cuhkigem2024@gmail.com </span>
                                        </a>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="divider"></div>
                        <div class="logo-container-wrapper">
                            <div> Affiliated with: </div>
                            <div class="cuhk-department-logo-container">
                                <a href="https://www.cuhk.edu.hk" target="_blank" rel="noopener noreferrer">
                                    <img src="/assets/logos/cuhk logo horizontal.svg" alt="CUHK logo" />
                                </a>
                                <a href="https://www.sls.cuhk.edu.hk" target="_blank" rel="noopener noreferrer">
                                    <img src="/assets/logos/sls logo.svg" alt="CUHK SLS logo" />
                                </a>
                                <a href="https://www.erg.cuhk.edu.hk" target="_blank" rel="noopener noreferrer">
                                    <img src="/assets/logos/foe logo.svg" alt="CUHK FoE logo" />
                                </a>
                                <a href="https://www.bme.cuhk.edu.hk" target="_blank" rel="noopener noreferrer">
                                    <img src="/assets/logos/bme logo.svg" alt="CUHK BME logo" />
                                </a>
                            </div>
                            <div> Sponsored by: </div>
                            <div class="sponsor-logo-container">
                                <a href="https://opentrons.com" target="_blank" rel="noopener noreferrer">
                                    <img src="/assets/logos/opentrons logo.svg" alt="Opentrons logo" />
                                </a>
                                <a href="https://www.idtdna.com" target="_blank" rel="noopener noreferrer">
                                    <img src="/assets/logos/idt logo.svg" alt="IDT logo" />
                                </a>
                                <a href="https://www.neb.com" target="_blank" rel="noopener noreferrer">
                                    <img src="/assets/logos/neb logo.svg" alt="NEB logo" />
                                </a>
                                <a href="https://www.hklss.org" target="_blank" rel="noopener noreferrer">
                                    <img src="/assets/logos/hklss logo repacked.svg" alt="HKLSS logo" />
                                </a>
                                <a href="https://abehk.fed.cuhk.edu.hk" target="_blank" rel="noopener noreferrer">
                                    <img src="/assets/logos/abehk logo.svg" alt="ABEHK logo" />
                                </a>
                            </div>
                        </div>
                    </div>
                    <div class="footer-persistent">
                        <div class="repository-container">
                            <!-- Repository links -->
                            <span>
                                Repository:
                                <a href="https://github.com/ResiSense/ResiSense.github.io" target="_blank">
                                    <img src="/assets/icons/github icon.svg" alt="GitHub icon" />
                                </a>
                                <a href="https://gitlab.igem.org/2024/hongkong-cuhk" target="_blank">
                                    <img src="/assets/icons/gitlab icon.svg" alt="GitLab icon" />
                                </a>
                            </span>
                        </div>
                        <div class="license-container">
                            <!-- Licensing https://creativecommons.org/licenses/by/4.0/ -->
                            <span>
                                ¬©Ô∏è 2024 ResiSense. Licensed under
                                <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">
                                    <img
                                        src="/assets/icons/creative common/Creative Common CC icon.svg"
                                        alt="Creative Commons icon"
                                    />
                                    <img
                                        src="/assets/icons/creative common/Creative Common BY icon.svg"
                                        alt="Creative Commons BY icon"
                                    />
                                    CC BY 4.0</a
                                >. Some rights reserved.
                            </span>
                        </div>
                    </div>
                </footer>
            </painted-footer>
        </painted-template>
    </body></html
>
